security/firewall_restriction.rst

```diff
@@ -1,8 +1,8 @@
 .. index::
    single: Security; Restrict Security Firewalls to a Request
 
-How to Restrict Firewalls to a Specific Request
-===============================================
+How to Restrict Firewalls by Pattern, Host or HTTP Method
```

Comment:
Instead of adding these details (*"by pattern, host or HTTP method"*) ... why not make this heading more generic: *"How to Restrict Firewalls to a Request"*